tests/Makefile

```diff
@@ -9,7 +9,7 @@ default: it
 
 .PHONY: clean default it test
 
-TESTBINS = unittest_stralloc
+TESTBINS = unittest_byte unittest_str unittest_stralloc
```

Comment:
Would be good to put this commit right after the .gitignore one (possibly squashing), so the tests are in place before changing anything.

Reply:
ignore what GitHub says about the order, look at git log.